flake.lock

```diff
@@ -129,6 +129,21 @@
         "type": "github"
```

Comment:
Changes to `flake.lock` are caused by the fact that `std` was updated and that the hard-coded revision hash was removed from the flake inputs urls.